test-charm/requirements.txt

```diff
@@ -1 +1 @@
-ops<1.3.0
+ops<1.3.0
```

Comment:
Can this file get reverted? Looks like the newline got stripped

Reply:
FIxed